### PR TITLE
Create an interactive plot tool for spectrograms

### PR DESCRIPTION
**Pull request by jibarnum**
Created the request 2D interactive plot to appear whenever there are spectrograms. I made it such that if you call tplot() separately from the cdf_to_tplot function, then you can add in interactive=True to make the extra 2D plot happen.

### CODE REVIEW

**Comment by bryan-harter** [comment]
Looks great on the spectrograms, but if I plot a spectrogram with a line plot, and then I hover over the line plot, the program freezes

**Comment by jibarnum** [comment]
Hey @bryan-harter , tell me with the updated code if that freezing issue with the program happens again. Also, updated for the logarithmic axis stuff if that's a thing in the spectrogram plots.

File: pytplot/Interactive2DPlot.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    z_values = list()
+    for r, rows in pytplot.data_quants[var].data.iterrows():
+        # r = time
+        # rows = the flux at each time, where each row signifies a different time
+        time_values.append(r)
+        z_values.append(rows.values)
+    return time_values, z_values
+
+
+def Interactive2DPlot():
```
Comment (bryan-harter) [inline pytplot/Interactive2DPlot.py]: Usually function names are are not capitalized, that is reserved for classes:

https://www.python.org/dev/peps/pep-0008/#class-names

I know it sounds nit picky, but this convention is actually used everywhere.

Comment (jibarnum) [reply pytplot/Interactive2DPlot.py]: Nah, that's fair, I'll be sure to change it.

**Pull request merged**